app/src/controller/inputcontroller.cpp

```diff
@@ -0,0 +1,24 @@
+#include "inputcontroller.h"
+
+#include <iostream>
+
+int InputController::InputProcess()
+{
+    int _choice;
+    std::cin >> _choice;
+    while (true)
+    {
+        if (std::cin.fail())
```

Comment:
Don't use an infinite loop. Write it like: `while (!std::cin.fail())
{
    ...
}`